views/rock.erb

```diff
@@ -0,0 +1,19 @@
+
+<h1 style="font-family: monospace; text-align: center"> <%= @game.player.name %>, you chose ROCK! </h1>
+<br>
+<h1 style="font-family: monospace; text-align: center"> Your oponent chose <%= @game.opponent_choice %>! </h1>
+<br>
+<% if @game.opponent_choice == "ROCK" %>
```

Comment:
Ideally this kind of logic should sit in the model layer instead (your class `Game, for example). There shouldn't be any code in the views/templates "deciding" what the result of the game should be, its only responsibility should be to display that result to the user - in that case you could have a method `game.result`, get the return value from the controller and use it in this view to display the appropriate content :)